### ISSUE TITLE
Repo vs. script?

### ISSUE THREAD

**Issue by cranmer** [uwescience/shablona#33]
Perhaps this isn't appropriate as an issue, but the model here is to clone the repo and do various renaming operations. An alternative strategy is to have Shablona be a collection of scripts that prepare the template like `shablona --package-name smallish --doc sphynx` or something like that.

**Comment by arokem** [comment]
Yes! @joseph-hellerstein made a similar comment the other day. I would like to automate this a bit more, but I actually like having the template itself available as a reference.. Also, what you propose here is very nicely handled by https://github.com/audreyr/cookiecutter, which does something like what you propose.
